EPMRPP-62670: Upload a .jar file and view new tabs

Alexandr-Kislov opened the pull request:
https://jiraeu.epam.com/browse/EPMRPP-62670

DESCRIPTION:
1) Added plugin-example project
2) Made some changes to define the plugin

VyacheslavGarshin commented:
description of PR shouldn't repeat the task itself=)
it's better to describe what exactly in the MR.
example: 1) added some projects 2) created some sql scripts 3) added several functions... etc.
several main changes would be nice!

Alexandr-Kislov commented:
> description of PR shouldn't repeat the task itself=)
> it's better to describe what exactly in the MR.
> example: 1) added some projects 2) created some sql scripts 3) added several functions... etc.
> several main changes would be nice!

Changed the description of the PR

Alexandr-Kislov commented on src/main/resources/schema/001_schema_init.sql:
```diff
@@ -0,0 +1,11 @@
+CREATE SCHEMA IF NOT EXISTS example;
```
Changed it in the scope of this [PR](https://github.com/reportportal/plugin-bts-azure/pull/3)

Alexandr-Kislov commented on src/main/java/com/epam/reportportal/extension/example/ExampleExtension.java:
```diff
@@ -0,0 +1,186 @@
+package com.epam.reportportal.extension.example;
```
Changed it in the scope of this [PR](https://github.com/reportportal/plugin-bts-azure/pull/3)

Alexandr-Kislov commented on src/main/resources/schema/001_schema_init.sql:
```diff
@@ -0,0 +1,11 @@
+CREATE SCHEMA IF NOT EXISTS example;
+
+CREATE TABLE IF NOT EXISTS example.entity
```
All the basic entities we save in the core schema. This entity is just an example for a situation if we will need some specific entities for our plugin, which we will save in our schema.
If it doesn't come in handy in the future I'll just delete it.
